app/views/users/show.html.slim

```diff
@@ -0,0 +1,42 @@
+h2 #{@user.first_name} #{@user.last_name}
+.user-profile.row 
+  .user-photo.col-md-3.col-xs-12
+      img src="#{@user.image_url}"
+  .user-info.col-md-9.col-xs-12
+    p Електронна пошта: #{@user.email}
+    p Запис зареєстровано: #{@user.created_at.strftime("%d-%m-%Y")}
+- if @issues.any?
+  .box.user-issues.col-xs-12
+    h4 Мої звернення:
+    .table-responsive
+      table.table.table-hover
+        thead
+          tr
+            th Звернення
+            th Назва
+            th Дата створення
+            th Опис
+            th Статус
+            th Посилання
+
+        tbody
+          - @issues.each do |issue|
+            tr
+              td № #{issue.id}
+              td =issue.title
```

Comment:
Also i recommend you to cut issue name and description.
Maybe it better to show on this page only title, status and link to issue? or something else. Not all content of issue.